client/containers/Pub/SpubHeader/spubHeader.scss

```diff
@@ -1,72 +1,24 @@
-@import '../PubHeader/themes.scss';
-@import 'styles/variables.scss';
-$light-grey: #BDBDBD;
-$lighter-grey: #747474;
-// $spub-configuration-width: 1000px;
-// $title-elements-spacing: 15px;
+@import '../PubHeader/pubHeader.scss';
+@import './styles.scss';
 
 .spub-header-component {
-	&,.minimal-editor-component > .editor-wrapper {
```

Comment:
I moved all of this stuff to the `SpubHeaderTab` component so it wouldn't affect the `PubHeader`.